src/libexpr/primops/fetchGit.cc

```diff
@@ -216,6 +238,8 @@ static void prim_fetchGit(EvalState & state, const Pos & pos, Value * * args, Va
                 rev = state.forceStringNoCtx(*attr.value, *attr.pos);
             else if (n == "name")
                 name = state.forceStringNoCtx(*attr.value, *attr.pos);
+            else if (n == "fetchSubmodules")
```

Comment:
Maybe rename this to `submodules`? "fetch" is superfluous since the function is already called `fetchGit`.

Reply:
Sounds good.

Reply:
Done.